extensions/pointcloud/json-schema/schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.stacspec.org/dev/extensions/pointcloud/json-schema/schema.json#",
+  "title": "Point Cloud Extension",
+  "description": "STAC Point Cloud Extension to a STAC Item",
+  "allOf": [
+    {
+      "$ref": "../../../item-spec/json-schema/item.json"
+    },
+    {
+      "$ref": "#/definitions/stac_extensions"
+    },
+    {
+      "$ref": "#/definitions/pointcloud"
+    }
+  ],
+  "definitions": {
+    "pointcloud": {
+      "type": "object",
+      "required": [
+        "stac_extensions",
+        "properties"
+      ],
+      "properties": {
+        "stac_extensions": {
+          "type": "array",
+          "contains": {
+            "const": "pointcloud"
+          }
+        },
+        "properties": {
+          "type": "object",
+          "required": [
+            "pc:count",
+            "pc:type",
+            "pc:encoding",
+            "pc:schemas"
+          ],
+          "properties": {
+            "pc:count": {
+              "type": "number",
+              "multipleOf": 1.0
+            },
+            "pc:type": { "type": "string" },
+            "pc:encoding": { "type": "string" },
```

Comment:
Not PR related, but just wondering what this field is about. Is this a media/mime type and better be part of the assets?

Reply:
I think `pc:type` is meant to specify how the pointcloud was generated: eopc, lidar, radar, or sonar. And `pc:encoding` would be the format: `LASzip`, `pbf`, `ept`, etc. However, the readme and `pdal-to-stac.py` differ slightly (python script only sets whether compression was used).

Based on the `example-autzen,json` I don't think these were meant to be assets. Perhaps renaming them:
`pc:type` -> `pc:device`
`pc:encoding` -> `pc:driver`
would avoid confusion?

Reply:
Okay, thanks. For compatibility, I'm fine with leaving it as it is and could just be improved a little in the documentation, but also not really related to this PR.

Reply:
I'll leave the renaming to another issue/PR.